app/screens/home/channel_list/categories_list/categories/body/index.ts

```diff
@@ -57,7 +57,8 @@ const observeCategoryChannels = (category: CategoryModel) => {
 };
 
 const enhanced = withObservables([], ({category, currentUserId, database, isTablet, locale}: EnhanceProps) => {
-    const channelsWithMyChannel = observeCategoryChannels(category);
+    const categoryMyChannels = category.myChannels.observeWithColumns(['last_post_at', 'is_unread']);
```

Comment:
This same observer is used here and in observeCategoryChannels. This way we only create it once, and we don't have to use the map that appears later.